invokeai/app/services/image_records/image_records_sqlite.py

```diff
@@ -211,6 +212,13 @@ def get_many(
                 """
                 query_params.append(board_id)
 
+            # Search term condition
+            if search_term:
+                query_conditions += """--sql
+                AND json_extract(images.metadata, '$') LIKE ?
```

Comment:
If we are just doing free text search, there's no need to use `json_extract`. Can just be

```sql
AND images.metadata LIKE ?
```